### ISSUE TITLE
Is it necessary to use "paired" data for training when eliminating the adversarial loss of GAN?

### ISSUE THREAD

**Issue by crissallan** [DeepMotionEditing/deep-motion-editing#48]
Hi Peizhuo,

Thanks for the guidance these days, I've reproduced part of your works.()(https://github.com/crissallan/Skeleton_Aware_Networks_for_Deep_Motion_Retargeting)

However, I haven't add the GAN part to the model(since training GAN needs lots of tricks). My training pipeline are now working in a "paired" way. Which means the .bvh file of skeleton A and B should be the same during every training iteration.

I tried to train the model in an "Un-paired"  way, but the loss is really hard to converge. According to the ablation study of your paper, it states that "omitting the adversarial loss improves the performance for intra-structural skeletons"

So I'd like to know that after deprecating the adv_loss, did you train the network in a "paired"(supervised) way or still in a "Un-paired" way?

Many thanks!

**Comment by PeizhuoLi** [comment]
You are welcome and thank you for your interest in our project!

Regarding your question, we always use the "unpaired" way, no matter whether there is "adv_loss". We find adv_loss is not crucial and it's mainly for better details, but the combination of all other loss plays an important role.

From my experience, training intra-structure retargeting using only one auto-encoder is not difficult, as long as it performs well on auto-encoding mission. However, training between two auto-encoders are much more tricky, thus I'd say each loss and the hyper-parameter will be very helpful.

Hope this will help.
